plugins/woocommerce/package.json

```diff
@@ -723,7 +723,6 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
-				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
❓ Why does this line get removed?

I checked out the repo and I ran `pnpm install` at the repo root, and I see this line gets added, see screenshot below:

![Screenshot 2024-07-30 at 8 38 21 PM](https://github.com/user-attachments/assets/c7bde5da-7a79-477b-beb6-5c551d5a9ec9)

So it seems that we should not remove this line?

Reply:
@ecgan this is a bug in the script that autogenerates these. @Saggre it would be best to revert the change to this line.

Reply:
@ecgan @samueljseay I restored the line